src/main/features/dashboard/views/index.njk

```diff
@@ -23,39 +23,39 @@
           <th scope="col">{{ t('Defendant name') }}</th>
           <th scope="col">{{ t('Claim amount') }}</th>
 
-          <th scope="col" class="mobile-hide">{{ t('Status') }}</th>
+          <th scope="col" class="phone-hide">{{ t('Status') }}</th>
```

Comment:
this should remain as mobile-hide as it relates to hiding this when a user in on a mobile